src/opf/dataset.py

```diff
@@ -238,7 +254,11 @@ def prepare_data(self):
 
     @property
     def case_path(self):
-        return Path(self.data_dir / f"{self.case_name}.json")
+        # TODO: eventually move this to front end
+        if self.case_name == "case118_ieee":
```

Comment:
Is this logic necessary? Can't I simply provide case_name `case118_ieee_removed_branch`?